sdk_contrib/async_processor/src/main/java/io/opentelemetry/sdk/contrib/trace/export/DisruptorEventQueue.java

```diff
@@ -92,8 +99,12 @@ public DisruptorEvent newInstance() {
     this.blocking = blocking;
   }
 
-  // Enqueues an event on the {@link DisruptorEventQueue}.
   void enqueue(ReadableSpan readableSpan, EventType eventType) {
+    enqueue(readableSpan, eventType, null);
+  }
+
+  // Enqueues an event on the {@link DisruptorEventQueue}.
+  void enqueue(ReadableSpan readableSpan, EventType eventType, CountDownLatch flushLatch) {
```

Comment:
Now that we pass a CDL for flush I think we should do the same for the shutdown one and not have a special case for that one.

Reply:
Make sense!